tmt/base.py

```diff
@@ -1991,15 +1995,15 @@ def create(
         # Override template with data provided on command line
         plan_content = Plan.edit_template(plan_content)
 
-        for name in names:
-            (directory, plan) = os.path.split(name)
-            directory_path = path / directory.lstrip('/')
-            has_fmf_ext = os.path.splitext(plan)[1] == '.fmf'
-            plan_path = directory_path / (plan + ('' if has_fmf_ext else '.fmf'))
+        for plan_name in names:
+            plan_path = Path(plan_name)
+
+            if plan_path.suffix != '.fmf':
+                plan_path = plan_path.parent / f'{plan_path.name}.fmf'
 
             # Create directory & plan
             tmt.utils.create_directory(
-                path=directory_path,
+                path=plan_path.parent,
```

Comment:
Maybe here is missing `path/` as you did in story

Reply:
Addressed in b70de6ef